### PR TITLE
Replace symlink with an actual copy of the env file in the kube base

### LINKED ISSUE #193: Kustomize doesn't support --load-restrictor LoadRestrictionsNone for remote resources
**Describe the bug**
When trying to follow [these](https://github.com/faros-ai/faros-community-edition/blob/main/kube/README.md) instructions for deploying FarosCE in Kubernetes I'm getting the following issue:

```
Error: accumulating resources: accumulation err='accumulating resources from './faros': '/private/var/folders/vx/gbhv_rk12vq1zdf76bpn0nhh0000gq/T/kustomize-1233669672/kube/base/faros' must resolve to a file': recursed accumulation of path '/private/var/folders/vx/gbhv_rk12vq1zdf76bpn0nhh0000gq/T/kustomize-1233669672/kube/base/faros': loading KV pairs: env source files: [./config/.env]: security; file '/private/var/folders/vx/gbhv_rk12vq1zdf76bpn0nhh0000gq/T/kustomize-1233669672/kube/base/faros/config/.env' is not in or below '/private/var/folders/vx/gbhv_rk12vq1zdf76bpn0nhh0000gq/T/kustomize-1233669672/kube/base/faros'
```    

This is caused by the fact that we are trying to re-utilize the existing `.env` configuration file in the root by symlinking it to `kube/faros/config/.env`. When kustomize parses this files, it attempts to load a resource that is above the path of the  `kustomization.yaml` file in question which is typically prohibited for security reasons. However this can be bypassed with the flag `--load-restrictor LoadRestrictionsNone` and that often works but as we have just found out it doesn't seem to hold true for when resources are being referenced remotely like the command above.

Here's a reference to a [kustomize Github issue](https://github.com/kubernetes-sigs/kustomize/issues/4052) touching on the unexpected behavior.

If one clones the repository and runs

```
kustomize build --load-restrictor LoadRestrictionsNone kube/base
```

things work properly, and the --load-restrictor flag is respected

**To Reproduce**
1. Install [kustomize](https://kubectl.docs.kubernetes.io/installation/kustomize/)
2. From a terminal execute `kustomize build --load-restrictor LoadRestrictionsNone https://github.com/faros-ai/faros-community-edition/kube/base`

**Expected behavior**
FarosCE K8s manifests are rendered correctly

**Screenshots and logs**
image

**Deployment**
* Kustomize 4.5.5

### PR DESCRIPTION
**Pull request by thomas-gerber**
# Description
Replace symlink with an actual copy of the env file in the kube base
until there is support for removing load restrictions in remote directories
see: https://github.com/kubernetes-sigs/kustomize/issues/4052

Fixes #193 

## Type of change
(Delete what does not apply)
- Bug fix (non-breaking change which fixes an issue)

# Checklist
(Delete what does not apply)
- [x] Have you checked to there aren't other open Pull Requests for the same update/change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you successfully run tests with your changes locally?

### CODE REVIEW

**Review by ypc-faros — APPROVED** [review]

**Review by tovbinm — COMMENTED** [review]
Why add `.env` file? It's gonna be pain to maintain both files.

**Comment by thomas-gerber** [comment]
> Why add `.env` file? It's gonna be pain to maintain both files.

See https://github.com/kubernetes-sigs/kustomize/issues/4052
We added an automated drift check in the workflows.

File: kube/README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ We provide all required Kubernetes manifests for a full stack deployment in this
 Provided you have have access to a Kubernetes cluster and have configured access to it via `kubectl`, FarosCE can be deployed with the following command:
 
 ```base
-kustomize build --load-restrictor LoadRestrictionsNone https://github.com/faros-ai/faros-community-edition/kube/base | kubectl apply -f -
+kubectl apply -k https://github.com/faros-ai/faros-community-edition/kube/base
 ```
 
 Kustomize will load the deployment definition in this repository over Github, build the manifests and pipe them over to `kubectl` for deployment in the configured cluster.
```
Comment (saviogl) [inline kube/README.md:13]:
```suggestion
Kustomize will load the deployment definition in this repository over Github, build the manifests and deploy in the configured cluster.
```

**Review by saviogl — APPROVED** [review]
👍

**Pull request merged**